system/system.proto

```diff
@@ -218,21 +220,22 @@ message PingResponse {
 // If the hop_count is -1 the traceroute will continue forever.
 //
 message TracerouteRequest {
-  string source = 1;         // Source address to ping from.
-  string destination = 2;    // Destination address to ping.
-  uint32 initial_ttl = 3;    // Initial TTL. (default=1)
-  int32 max_ttl = 4;         // Maximum number of hops. (default=30)
-  int64 wait = 5;            // Nanoseconds to wait for a response.
-  bool do_not_fragment = 6;  // Set the do not fragment bit. (IPv4 destinations)
-  bool do_not_resolve = 7;   // Do not try resolve the address returned.
-  types.L3Protocol l3protocol = 8; // Layer-3 protocol requested for the ping.
+  string source = 1;                // Source address to ping from.
+  string destination = 2;           // Destination address to ping.
+  uint32 initial_ttl = 3;           // Initial TTL. (default=1)
+  int32 max_ttl = 4;                // Maximum number of hops. (default=30)
+  int64 wait = 5;                   // Nanoseconds to wait for a response.
+  bool do_not_fragment = 6;         // Set the do not fragment bit. (IPv4 destinations)
+  bool do_not_resolve = 7;          // Do not try resolve the address returned.
+  types.L3Protocol l3protocol = 8;  // Layer-3 protocol requested for the ping.
```

Comment:
nit: which formatting/linter tool did you use to align the inline comments like that?
I used https://github.com/yoheimuta/protolint and it doesn't align comments that way

Reply:
I just used the spacebar, I'll remove it though.

Interestingly enough, running that linter against this file shows a-lot of failures well beyond the content added by this commit though.

```[gnoi\system\system.proto:131:1] The line length is 85, but it must be shorter than 80
[gnoi\system\system.proto:184:1] The line length is 86, but it must be shorter than 80
[gnoi\system\system.proto:185:1] The line length is 87, but it must be shorter than 80
[gnoi\system\system.proto:188:1] The line length is 93, but it must be shorter than 80
[gnoi\system\system.proto:228:1] The line length is 87, but it must be shorter than 80
[gnoi\system\system.proto:238:1] The line length is 93, but it must be shorter than 80
[gnoi\system\system.proto:275:1] The line length is 82, but it must be shorter than 80
 "REBOOT_METHOD"
[gnoi\system\system.proto:129:3] EnumField name "POWERDOWN" should have the prefix "REBOOT_METHOD"
[gnoi\system\system.proto:130:3] EnumField name "HALT" should have the prefix "REBOOT_METHOD"
[gnoi\system\system.proto:131:3] EnumField name "WARM" should have the prefix "REBOOT_METHOD"
[gnoi\system\system.proto:132:3] EnumField name "NSF" should have the prefix "REBOOT_METHOD"
[gnoi\system\system.proto:135:3] EnumField name "POWERUP" should have the prefix "REBOOT_METHOD"
[gnoi\system\system.proto:232:5] EnumField name "ICMP" should have the prefix "L4PROTOCOL"
[gnoi\system\system.proto:233:5] EnumField name "TCP" should have the prefix "L4PROTOCOL"
[gnoi\system\system.proto:234:5] EnumField name "UDP" should have the prefix "L4PROTOCOL"
[gnoi\system\system.proto:266:5] EnumField name "DEFAULT" should have the prefix "STATE"
[gnoi\system\system.proto:267:5] EnumField name "NONE" should have the prefix "STATE"
[gnoi\system\system.proto:268:5] EnumField name "UNKNOWN" should have the prefix "STATE"
[gnoi\system\system.proto:269:5] EnumField name "ICMP" should have the prefix "STATE"
[gnoi\system\system.proto:270:5] EnumField name "HOST_UNREACHABLE" should have the prefix "STATE"
[gnoi\system\system.proto:271:5] EnumField name "NETWORK_UNREACHABLE" should have the prefix "STATE"
[gnoi\system\system.proto:272:5] EnumField name "PROTOCOL_UNREACHABLE" should have the prefix "STATE"
[gnoi\system\system.proto:273:5] EnumField name "SOURCE_ROUTE_FAILED" should have the prefix "STATE"
[gnoi\system\system.proto:274:5] EnumField name "FRAGMENTATION_NEEDED" should have the prefix "STATE"
[gnoi\system\system.proto:275:5] EnumField name "PROHIBITED" should have the prefix "STATE"
[gnoi\system\system.proto:276:5] EnumField name "PRECEDENCE_VIOLATION" should have the prefix "STATE"
[gnoi\system\system.proto:277:5] EnumField name "PRECEDENCE_CUTOFF" should have the prefix "STATE"
[gnoi\system\system.proto:127:3] EnumField name "REBOOT_METHOD_UNKNOWN" with zero value should have the suffix "UNSPECIFIED"
[gnoi\system\system.proto:232:5] EnumField name "L4PROTOCOL_ICMP" with zero value should have the suffix "UNSPECIFIED"
[gnoi\system\system.proto:266:5] EnumField name "STATE_DEFAULT" with zero value should have the suffix "UNSPECIFIED"
[gnoi\system\system.proto:289:3] Repeated field name "as_path" must be pluralized name "as_paths"```